Content.Server/Drone/DroneSystem.cs

```diff
@@ -37,9 +41,9 @@ public override void Initialize()
             SubscribeLocalEvent<DroneComponent, ThrowAttemptEvent>(OnThrowAttempt);
         }
 
-        private void OnInteractionAttempt(EntityUid uid, DroneComponent component, InteractionAttemptEvent args)
+        private async void OnInteractionAttempt(EntityUid uid, DroneComponent component, InteractionAttemptEvent args)
```

Comment:
why is this async

Reply:
my ide randomly inserts that for no reason and I have to hunt hem down usually